packages/nitro-protocol/test/contracts/NitroAdjudicator/concludePushOutcomeAndTransferAll.test.ts

```diff
@@ -173,6 +173,22 @@ describe('concludePushOutcomeAndTransferAll', () => {
           await Token.transfer(ERC20AssetHolder.address, BigNumber.from(heldBefore.ERC20.c))
         ).wait(); // if the tx is mined, we know we the transfer succeeded
       }
+      if ('ETH' in heldBefore) {
+        // Transfer some funds in to the ETHAssetHolder. Use zero channel address for ease.
+        await (
+          await EthAssetHolder1.deposit(channelId, '0x00', heldBefore.ETH.c, {
+            value: heldBefore.ETH.c,
+          })
+        ).wait();
+      }
+      if ('ETH2' in heldBefore) {
+        // Transfer some funds in to the other ETHAssetHolder. Use zero channel address for ease.
+        await (
+          await EthAssetHolder2.deposit(channelId, '0x00', heldBefore.ETH2.c, {
+            value: heldBefore.ETH2.c,
+          })
+        ).wait();
+      }
```

Comment:
Is this the change that makes means we are now "[using] a full eth asset holder in tests"?

Reply:
No, this is just another follow on change that is necessary, given that we now use the full eth asset holder in tests. You can't pay ETH out, unless you have paid some in :-)